pkg/openthread/Makefile

```diff
@@ -0,0 +1,54 @@
+PKG_NAME=openthread
+PKG_URL=https://github.com/openthread/openthread.git
+PKG_VERSION=fbfd76a990b81f007957e1bd774e51bce742e53e
+PKG_BUILDDIR ?= $(BINDIRBASE)/pkg/$(BOARD)/$(PKG_NAME)
+
+ifneq (,$(filter libopenthread-ftd,$(USEMODULE)))
+    $(info Compile OpenThread for FTD device)
+    OPENTHREAD_ARGS+= --enable-cli-app=ftd
+endif
+ifneq (,$(filter libopenthread-mtd,$(USEMODULE)))
+    $(info Compile OpenThread for MTD device)
+    OPENTHREAD_ARGS+= --enable-cli-app=mtd
+endif
+ifneq (,$(filter libopenthread-ncp-ftd,$(USEMODULE)))
+    $(info Compile OpenThread with NCP)
+    OPENTHREAD_ARGS += --enable-ncp-app=ftd --with-ncp-bus=uart
+endif
+$(info $$OPENTHREAD_ARGS is [${OPENTHREAD_ARGS}])
+
+.PHONY: all
+
+all: git-download
+	cd $(PKG_BUILDDIR) && PREFIX="/" ./bootstrap
+	cd $(PKG_BUILDDIR) && ./configure \
+		--enable-application-coap INSTALL="/usr/bin/install -p" CPP="$(CPP)" \
```

Comment:
the INSTALL part shouldn't be needed.  I'm checking what happens if I remove that.